taxi/urls.py

```diff
@@ -2,25 +2,59 @@
 
 from .views import (
     index,
+    ManufacturerCreateView,
+    ManufacturerListView,
+    ManufacturerUpdateView,
+    ManufacturerDeleteView,
+    CarCreateView,
     CarListView,
+    CarUpdateView,
+    CarDeleteView,
     CarDetailView,
     DriverListView,
-    DriverDetailView,
-    ManufacturerListView,
+    DriverDetailView
 )
 
 urlpatterns = [
     path("", index, name="index"),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create"
+    ),
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
-        name="manufacturer-list",
+        name="manufacturer-list"
+    ),
+    path(
+        "manufacturers/pk/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update"
+    ),
+    path(
+        "manufacturers/pk/<int:pk>/delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete"
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
-    path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/pk/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
```

Comment:
without pk was correct, fix it also in other similar cases

Reply:
But with pk is correct to, isn't it? i will refactoring code if you need